personalized_nlp/learning/train.py

```diff
@@ -25,10 +25,10 @@ def train_test(datamodule, model, epochs=6, lr=1e-2, weight_decay=0.0, regressio
             class_names = [datamodule.annotation_column]
 
         if not model._frozen and weight_decay!=0.0:
-          model = RegressorFinetune(model=model, lr=lr, class_names=class_names)
+          model = RegressorFinetune(model=model, lr=lr, class_names=class_names, nr_frozen_epochs=nr_frozen_epochs)
 
         model = Regressor(model=model, lr=lr,
```

Comment:
Shouldn't it be here some `else`, `model` will not be overwritten by `Regressor` object even if it was at first `RegressorFinetune`?

Reply:
Yes I've resolved that problem in this fix also. Please check it